app/components/modal/modal.css

```diff
@@ -16,6 +16,8 @@
 
   background: rgba(0, 0, 0, 0.4);
   backdrop-filter: blur(1px) saturate(25%);
+
+  z-index: 1000;
```

Comment:
the war begins

Reply:
Lol, as I was writing this I had a big sigh and went "let the games begin"